net/haproxy/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		haproxy
-PLUGIN_VERSION=		2.7
+PLUGIN_VERSION=		2.8
```

Comment:
I'm sorry wtf?

1. don't bump the version, bump the revision
2. this is haproxy, not acme-client

Reply:
ps: when there is a revision, you cannot bump the version without clearing the revision... it's pretty simple.

Reply:
not good, will fix

Reply:
I already did

Reply:
#755 includes the fix - I should not do the merges that late…